### ISSUE TITLE
Apator WM-Bus-08 FLOW water

### ISSUE THREAD

**Issue by DybaMati** [weetmuts/wmbusmeters#507]
Hello,
Does anyone know if the apator 08 overlay has information about the current m3/h or l/h water flow ??

**Comment by Santa77** [comment]
Nope, apator 08 via wmbusmetters show only total consumption of m3. But it sneds data every 2-3 minutes (except in night, during night is silent). So u can compute average consumprion between 2 frames, 2 packets...
